api/checks/webhook.go

```diff
@@ -143,6 +169,8 @@ func handleCheckSuiteEvent(aeAPI shared.AppEngineAPI, checksAPI API, taskcluster
 		if action == "requested" {
 			// For new suites, check if the pull is across forks; if so, request a suite
 			// on the main repo (web-platform-tests/wpt) too.
+			//
```

Comment:
If I'm reading this right we're inside `handleCheckSuiteEvent`, so this seems circular and not so much redundant. Whatever it is, it could do with an explanation :)